packages/api/app/index.js

```diff
@@ -70,7 +70,7 @@ app.use(boom());
 app.use(morgan('combined'));
 app.use(cors());
 app.use(cookieParser());
-app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.urlencoded({ limit: '1mb', extended: true }));
```

Comment:
Should we make this limit configurable? The current default (when you don't specify a limit) is 100kb.

Reply:
Any input on a reasonable number for the limit here would be much appreciated as well.

Reply:
I suggest to set it to 6MB which is the lambda payload limit.

Reply:
https://github.com/nasa/cumulus/pull/3122/commits/19b5ce631e65643f6f351997eea2a31242adb653